Source/ModelGroup.cs

```diff
@@ -7,14 +7,14 @@ namespace AssetGenerator
 {
     internal abstract partial class ModelGroup
     {
-        internal abstract ModelGroupName Name { get; }
-        internal List<Model> Models { get; set; }
-        internal List<Property> CommonProperties { get; private set; }
-        internal List<Property> Properties { get; private set; }
-        internal List<Runtime.Image> UsedTextures { get; private set; }
-        internal List<Runtime.Image> UsedFigures { get; private set; }
-        internal int Id { get; set; }
-        internal readonly bool NoSampleImages;
+        public abstract ModelGroupName Name { get; }
+        public List<Model> Models { get; set; }
```

Comment:
This should be `private set` also

Reply:
It gives an error when we set `this.Models` in the child classes
"The property or indexer 'ModelGroup.Models' cannot be used in this context because the set accessor is inaccessible"

Seems weird though, I would have expected it to work as private.

Reply:
Aah, right. It should be `protected` not `private`.

Reply:
Ok, it works if set to `protected`, due to being a derived class. That ok with you?

Reply:
Made the change

Reply:
Yup.

Reply:
Is there anything else that should be changed?

Reply:
Just waiting for the `PostRuntimeChanges` fix